### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by arnemauer** [squishykid/solax#79] [General]
Hi squishykid,

When can we expect a new version with the commits of 23/25 april and 18 july?

**Comment by squishykid** [comment]
should be out now!

I made a release here: https://github.com/squishykid/solax/releases/tag/v0.3.0
Which was included in home-assistant core here: https://github.com/home-assistant/core/pull/78219
And then released in home-assistant 2022.10: https://www.home-assistant.io/changelogs/core-2022.10

**Comment by vasimi** [comment]
Hi @squishykid, do you have any plans to make a new release? Support for X3 Hybrid G4 (introduced in #100, merged more than a year ago) is still missing in the latest release.

thx
